CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Univesal padding setted on body.
+- Class force-full-width to ignore body padding when its needed.
+- Class vtex-page-pd that behaves the same as body padding.
```

Comment:
why we need this vtex-page-pd? 🤔

Reply:
When you have a forced full width element and want to the internal content follow the body padding.